### PR TITLE
Fill coverage with total area and other enhancements

### PR DESCRIPTION
**Pull request by avancinirodrigo**
- [x] #824 
- [x] #2261
- [x] #1994 
- [x] #2263 
- [x] #2237 
- [x] #2068
- [x] #2172 
- [x] #1868

### CODE REVIEW

**Comment by avancinirodrigo** [comment]
@pedro-andrade-inpe 
Give a look at execution tests.
Shall I set progress 'false' to the projects scripts?

**Comment by pedro-andrade-inpe** [comment]
Yes.

File: packages/gis/lua/Layer.lua [hunk 1]
```diff
@@ -593,6 +605,7 @@ Layer_ = {
 	-- with each new layer's name formed by the own Layer's name and the respective times as sufix.
 	-- The default value is false and, in this case, the temporal data will be filled in the own Layer
 	-- into different attributes though.
+	-- @arg data.progress A boolean to set the progress viewer of the filling. Its default is true.
```
Comment (pedro-andrade-inpe) [inline packages/gis/lua/Layer.lua]: A boolean value indicating whether progress will be shown while computing the operation. The default value is true.

File: packages/gis/lua/Layer.lua [hunk 2]
```diff
@@ -534,51 +545,52 @@ Layer_ = {
...
-	-- & attribute, layer & dummy, pixel, split \
-	-- "coverage" & Percentage of each qualitative value covering the cell, using polygons or
+	-- & attribute, layer & dummy, pixel, progress, split \
+	-- "coverage" & Percentage or total area of each qualitative value covering the cell, using polygons or
 	-- raster data. It creates one new attribute for each available value, in the form
 	-- attribute.."_"..value, where attribute is the value passed as argument to fill and
 	-- value represent the different values in the input data.
 	-- The sum of the created attribute values for a given cell will range
-	-- from zero to one, according to the area of the cell covered by pixels.
+	-- from zero to one (default) or the total area (area = true), according to the area of the
```
Comment (pedro-andrade-inpe) [inline packages/gis/lua/Layer.lua]: from zero to one indicating the percentage of the cell covered by pixels of each class (default), or the total area covered by each class, according to the area of a pixel in the raster data (when using area = true).

**Pull request merged**